Bump `MLflow` to 2.2.2 (#1441)
zenml-io/zenml: docs/book/component-gallery/experiment-trackers/mlflow.md

[authentication related parameters](#authentication-methods) to be configured
for the MLflow Experiment Tracker.

{% hint style="warning" %}

> Due to a found [Vulnerability](https://github.com/advisories/GHSA-xg73-94fp-g449) in older versions of MLflow, we recommend using
> MLflow version 2.2.1 or higher for this scenario.

Reviewer: ```suggestion
Due to a [critical severity vulnerability](https://github.com/advisories/GHSA-xg73-94fp-g449) found in older versions of MLflow, we recommend using
MLflow version 2.2.1 or higher.
```
